apps/evm/src/types/index.ts

```diff
@@ -1,8 +1,12 @@
 import type { Token as PSToken } from '@pancakeswap/sdk';
+import type { ChainId } from '@venusprotocol/chains';
 import type BigNumber from 'bignumber.js';
 import type { BaseContract, ContractReceipt } from 'ethers';
 import type { TransactionReceipt } from 'viem';
 
+// TODO: import from package in places where it's used in the codebase
+export { ChainId, type ChainMetadata } from '@venusprotocol/chains/types';
```

Comment:
This code is here temporarily to prevent polluting this PR with hundreds of import changes. As more reusable packages will be created, I think it would be a good practice to focus on keeping PRs creating them as small as possible by re-exporting types used across the codebase. Once the most significant reusable packages created we can open new PRs to remove these re-exports.